.github/workflows/drc_io_sram_test.yml

```diff
@@ -0,0 +1,68 @@
+# Copyright 2022 GlobalFoundries PDK Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This is a Github Actions for testing DRC rule deck of GF180MCU
+name: DRC IO and SRAM test
+
+# Controls when the workflow will run
+on:
+  # Triggers the workflow on push or closed pull request events
+  push:
+  pull_request:
+    types:
+      - closed
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+# Defines environment variables
+env:
+  PDK_ROOT: /home/runner/work/globalfoundries-pdk-libs-gf180mcu_fd_pr/globalfoundries-pdk-libs-gf180mcu_fd_pr/rules/klayout
+  PDK: drc
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "DRC_testing"
+  DRC_IO_and_SRAM_testing:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v3
+
+      # Gets python version with list of needed requirements
+      - uses: actions/setup-python@v3
+        with:
+          python-version: '3.9'
+          cache: 'pip'
+          cache-dependency-path: '**/requirements_drc.txt'
+
+      # Installs requirements
+      - run: pip install -r ./rules/klayout/drc/testing/requirements_drc.txt
+
+      # Installs Klayout verison 0.27.8
+      - name: Installing Klayout
```

Comment:
Can we use KLayout from PyPi instead? See;
 * https://pypi.org/project/klayout/
 * https://www.klayout.org/klayout-pypi/

Reply:
Unfortunately we can't do that. Because Klayout installed from PyPi doesn't include DRC engine to run DRC scripts.

Reply:
@proppy / @joamatab - We should figure out how to get DRC engine into the KLayout installed from PyPi.